js_modules/dagit/packages/core/src/workspace/asset-graph/AssetNode.tsx

```diff
@@ -243,6 +244,18 @@ export const ASSET_NODE_FRAGMENT = gql`
   fragment AssetNodeFragment on AssetNode {
     id
     opName
+    op {
+      name
+      description
+      outputDefinitions {
```

Comment:
Lets merge this PR before the other global asset graph stuff - I think we're also fetching asset + ops together there and we might need to merge the two sets of query changes!

Reply:
Think I addressed your comments, and can merge when you're ready, but pls give final confirmation (especially on `TableMetadata` thing).

Reply:
Thanks! Will take a  look in a few minutes 👍